### PR TITLE
Updates unit tests to work with newer version of nock

### PR DESCRIPTION
**Pull request by dcalvoalonso**

### CODE REVIEW

File: lib/plugins/attributeAlias.js [hunk 1]
```diff
@@ -38,7 +38,10 @@ function extractSingleMapping(previous, current) {
 
     previous.direct[current.object_id] = current.name;
```
Comment (fgalan) [inline lib/plugins/attributeAlias.js]: An entry in the CNR file about this fix should be added

Comment (dcalvoalonso) [reply lib/plugins/attributeAlias.js]: Fixed in 06f744cfe57ae38e40bd57a84028f20a40bf0e53

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1,2 +1,3 @@
 Add protocol in device provision if not provided (#652)
-Fix: timestamp is now included just in non empty (more than id and type) multienty entitites
\ No newline at end of file
+Fix: timestamp is now included just in non empty (more than id and type) multienty entitites
+Fix: object_id fields are not introduced in CB requests by alias plugon (#660)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: plugon? :)

Comment (dcalvoalonso) [reply CHANGES_NEXT_RELEASE]: Fixed with 2c67b6d7908ceae3e793ce993f99b73b678f3e14

**Pull request merged**